Frontend/src/Components/utils/likeStatusUtil.js

```diff
@@ -0,0 +1,24 @@
+const API_KEY = import.meta.env.VITE_BACKEND_ADDRESS;
+
+export const fetchListingsWithStatusAndLiked = async (listings) => {
```

Comment:
is there any scenario when fetching listings that we don't want the status and liked data? if not, maybe we could do this logic on the back-end before returning listings in each get listings API call (the logic would be in a helper function as well)?

Reply:
i don't think there is a scenario where we do not require the status and liked data. All the places that call it, requires it